bundles/mapping/mapmodule/plugin/layers/LayerSelectionPlugin.js

```diff
@@ -37,6 +37,7 @@ Oskari.clazz.define('Oskari.mapframework.bundle.mapmodule.plugin.LayerSelectionP
             },
             buttonGroup: 'mobile-toolbar'
         };
+        me._styleSelectable = !!this.getConfig().isStyleSelectable;
```

Comment:
What's the difference with getConfig() and the config that is received as parameter? BasicMapModulePlugin handles setting the config that's returned by getConfig()?

Reply:
Yes, `config` param of constructor is assigned to instance variable `_config` by BasicMapModulePlugin. `getConfig()` reads that.